dipy/viz/fvtk.py

```diff
@@ -79,6 +79,9 @@
     msg = "Python VTK is not installed"
     warn(msg)
 
+    ren, have_ren, _ = optional_package('dipy.viz.window.ren',
```

Comment:
Reading this again -- do you understand why the line above this one throws a warning, instead of raising an error?